packages/release/activate-preconfigured.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Isolates preconfigured.target
+DefaultDependencies=no
+After=fipscheck.target
+Requires=fipscheck.target
```

Comment:
This is very similar to the existing `activate-configured.service` and `activate-multi-user.service`.